File: src/content/python/factor.py
num = int(input("Enter a number: "))
print(f"Prime factors of {num} are: ", end="")
i = 2
while i <= num:
    if num % i == 0:
        print(i, end=' ')
        num = num / i
    else:
        i += 1
print()

File: src/content/go/factor.go
package main

import (
    "fmt"
    "math"
)

func main() {
    var num int
    fmt.Print("Enter a number: ")
    fmt.Scan(&num)

    fmt.Printf("Prime factors of %d are: ", num)
    for i := 2; i <= int(math.Sqrt(float64(num))); i++ {
        for num%i == 0 {
            fmt.Printf("%d ", i)
            num /= i
        }
    }

    if num > 1 {
        fmt.Printf("%d ", num)
    }
	fmt.Printf("\n");
}